coordinator/app/src/main/kotlin/net/consensys/zkevm/coordinator/blockcreation/BlockCreationMonitor.kt

```diff
@@ -21,7 +20,7 @@ import kotlin.time.Duration.Companion.days
 
 class BlockCreationMonitor(
   private val vertx: Vertx,
-  private val extendedWeb3j: ExtendedWeb3J,
+  private val web3j: ExtendedWeb3J,
```

Comment:
Just to give more context. Once we switch to Beta V2 on Mainnet, there is no reason to monitor block creation, entrypoint for conflation would be trace counting API directly.

We didn't create new, traces-v2 alternative to BlockCreationMonitor to avoid excessive code branching in conflation flow, which is not trivial to follow already.